test/blackbox-tests/test-cases/watching/fs-memo.t

```diff
@@ -431,8 +431,11 @@ However, deleting [another-dir] isn't handled correctly.
 
   $ test "rm another-dir/file-7; sleep 0.001; rmdir another-dir"
   ------------------------------------------
+  Failure
   Success, waiting for filesystem changes...
-  Success, waiting for filesystem changes...
+  File "dir/file-7", line 1, characters 0-0:
+  Error: File unavailable: dir/file-7
+  Had errors, waiting for filesystem changes...
```

Comment:
Hah, interesting, this fixes an old CR-someday from me (see above). Was this expected?

Reply:
Not entirely, but I suppose it makes progress on the issue. Broken symlinks within the repo are still not well supported.

Reply:
Ah, I think I understand now. You've made some improvements around symbolic links in `cached_digest` earlier, and we're getting the benefits, but not in `fs_cache`. Makes sense I guess.